.github/ci_setup.sh

```diff
@@ -15,11 +15,14 @@ PLATFORM_NAME=$1
 
 # Set up the correct environment for the SYCL tests.
 if [ "${PLATFORM_NAME}" = "SYCL" ]; then
-   source /opt/intel/oneapi/setvars.sh --include-intel-llvm
+   if [ -f "/opt/intel/oneapi/setvars.sh" ]; then
+      source /opt/intel/oneapi/setvars.sh --include-intel-llvm
+   fi
    # Use clang/clang++ instead of icx/icpx, to avoid some aggressive math
    # optimizations that break some tests.
    export CC=`which clang`
    export CXX=`which clang++`
    export SYCLCXX="${CXX} -fsycl"
-   export SYCL_DEVICE_FILTER=host
+   export ONEAPI_DEVICE_SELECTOR=opencl:cpu
+   export SYCLFLAGS="-fsycl -fsycl-targets=spir64"
```

Comment:
Of course the problem is that now that you added NVIDIA and AMD backended build jobs, this setting mustn't be done here. Since right now we continue to only do Intel backed builds in the CI, with 3 different Docker images. 😛

Reply:
OK, can we do that in a separate PR maybe? Because I have a lot of work depending on this PR...